core/server/worker/src/main/java/alluxio/worker/page/PagedBlockMetaStore.java

```diff
@@ -217,8 +218,14 @@ public void addPage(PageId pageId, PageInfo pageInfo) {
     mDelegate.addPage(pageId, pageInfo);
   }
 
+  private boolean containsBlockMetaOfPage(PageId pageId) {
```

Comment:
Where is this method called?

Reply:
Although this method has not been called by any object, it would be useful in the future as we can check the status by calling this method. Thus, it is not necessary to remove this method.